libs/atqr/domain/src/lib/domain-spec/stories/judge/selectNewJudge.disl

```diff
@@ -0,0 +1,65 @@
+💬 SelectNewJudge! challenge: Challenge
+    NotificationService.SelectNewJudge(challenge)
+
+💬 SupervisionRequest! challenge: Challenge, contactJudge: Contact
+    NotificationService.SupervisionRequested(challenge, contactJudge)
+    
+
+
+service NotificationService
+
+    💬 SelectNewJudge! challenge: Challenge
+        sendEmail
+          template: SelectNewJudge
+          data: Challenge
+          to: challenge.player
+        when get the response   📣 supervisionRequest
+        if(pass X days send again, if pass Y days cancel)
```

Comment:
This has to be trigger by a time based (⏰) event (look for `ANewDayArrived` in `api.disl`)